content/terraform/v1.13.x/docs/language/meta-arguments/providers.mdx

```diff
@@ -0,0 +1,143 @@
+
+Specifying a `providers` argument cancels the default behavior. As a result, the child module only has access to the provider configurations you specify.
+
+
+### When to specify `providers`
+
+Add the `providers` argument when one of the following conditions apply:
+
+- Using different default provider configurations for a child module.
+- Configuring a module that requires multiple configurations of the same provider.
+
+### Change default provider configurations
+
+Most re-usable modules only use default provider configurations, which they can
+automatically inherit from their caller when `providers` is omitted.
+
+However, in Terraform configurations that use multiple configurations of the same provider, you might want some child modules to use the default provider configuration and others to use an alternate. 
+A common use case is using one configuration to manage resources in multiple different regions of the same cloud provider.
+
+Using `providers` argument lets you accommodate this without needing to edit the child module. Although the code
+within the child module always refers to the default provider configuration, the
+actual configuration of that default can be different for each instance.
+
+
+## Information for module developers
+
+For more details and guidance about working with providers inside a re-usable
+child module, refer to [Providers Within Modules](/terraform/language/modules/develop/providers).
+
+## Supported constructs
+
+You can use `providers` in the following Terraform configuration blocks:
+
+- [`module` blocks](/terraform/language/block/module)
+
+### Example use cases
+
+The following use cases describe common patterns for the `providers` argument.
+
+### Apply different provider configurations for module resources
+
+In the following example, the `tunnel` module includes resources that support their own provider configuration. As a result, each resource alias maps to a different provider configurations declared in the root module:
+
+```hcl
+provider "aws" {
+  alias  = "usw1"
+  region = "us-west-1"
+}
+
+provider "aws" {
+  alias  = "usw2"
+  region = "us-west-2"
+}
+
+module "tunnel" {
+  source    = "./tunnel"
+  providers = {
+    aws.src = aws.usw1
+    aws.dst = aws.usw2
+  }
+}
+```
+
+### Modules with alternate provider configurations
+
+In rare cases, a single re-usable module might require multiple configurations
+of the same provider. For example, a module that configures connectivity between
+networks in two AWS regions is likely to need both a `source` and a `destination`
+region. The following example shows possible root module configuration for this use case:
+
+```hcl
```

Comment:
Is this not the same example as above? I like this extra context in this copy of it though :)